crates/forge/tests/e2e/common/runner.rs

```diff
@@ -33,13 +33,35 @@ pub(crate) fn test_runner(temp_dir: &TempDir) -> SnapboxCommand {
 
 pub(crate) static BASE_FILE_PATTERNS: &[&str] = &["**/*.cairo", "**/*.toml"];
 
+fn is_package_from_docs_listings(package: &str) -> bool {
```

Comment:
You could [canonicalize](https://doc.rust-lang.org/beta/std/fs/fn.canonicalize.html) both "docs/listings/" directory and given package and check if dir `.starts_with`.

Reply:
I think we can just check if `canonicalize` returns `is_ok`, like this

```rust
fn is_package_from_docs_listings(package: &str) -> bool {
    let package_path = Path::new("../../docs/listings").join(package);
    fs::canonicalize(&package_path).is_ok()
}
```